contrib/mesos/pkg/controllermanager/controllermanager.go

```diff
@@ -167,14 +167,28 @@ func (s *CMServer) Run(_ []string) error {
 	namespaceController := namespacecontroller.NewNamespaceController(kubeClient, &unversioned.APIVersions{}, s.NamespaceSyncPeriod)
 	namespaceController.Run()
 
-	pvclaimBinder := persistentvolumecontroller.NewPersistentVolumeClaimBinder(kubeClient, s.PVClaimBinderSyncPeriod)
+	volumePlugins := app.ProbeRecyclableVolumePlugins(s.VolumeConfigFlags)
+	qosClasses := s.VolumeConfigFlags.ProvisionerQualityOfServiceClasses
+	provisioners, err := app.NewVolumeProvisioners(volumePlugins, qosClasses)
```

Comment:
Not your code, but i think the `kube-controller-manager/app` package should be aliased to something descriptive here.  How about `kubecontrollermanager`?